src/session/supergroup.rs

```diff
@@ -125,4 +126,32 @@ impl Supergroup {
         let self_ = imp::Supergroup::from_instance(self);
         self_.is_channel.get()
     }
+
+    pub fn member_count_expression(&self) -> gtk::Expression {
+        let supergroup_expression = gtk::ConstantExpression::new(self).upcast();
+        gtk::ClosureExpression::new(
+            |args| {
+                let group = args[1].get::<Supergroup>().unwrap();
+                let is_channel = group.is_channel();
+                let member_count = group.member_count();
+                if is_channel {
+                    if member_count > 1 {
+                        // Translators: <NUM> channel subscribers
+                        gettext!("{} subscribers", member_count)
+                    } else {
+                        gettext!("{} subscriber", member_count)
+                    }
+                } else if member_count > 1 {
+                    // Translators: <NUM> group members
+                    gettext!("{} members", member_count)
+                } else if member_count == 1 {
+                    gettext("1 member")
+                } else {
+                    gettext("group")
```

Comment:
Really not sure whether we should capitalize this. @melix99?

Reply:
I tried to set this strings via GTK Inspector to AdwHeaderBar::subtitle and it looks a bit weird with sentence or title capitalization. I think it's better having lowercase strings like it is now.